bundle/src/main/java/com/adobe/acs/commons/fam/impl/ActionManagerImpl.java

```diff
@@ -278,9 +278,10 @@ private void deferredWithResolver(
                 if (!closesResolver) {
                     logCompletetion();
                 }
-            } catch (Exception ex) {
+            } catch (Throwable t) {
```

Comment:
I see -- well point taken.  I still want to know if any failure occurs but the RuntimeException bit might be a better idea.  I was having trouble with it catching checked exceptions correctly.